Source/FullScreenPokemon.d.ts

```diff
@@ -1093,6 +1108,11 @@ declare module FullScreenPokemon {
          * A Function to be called when the item is used.
          */
         bagActivate?: Function;
+
+        /**
+         * The name of the item.
+         */
+        title?: String;
```

Comment:
Also, this would only make sense if you added a `.title` to every item schema. That's a lot of work.

Instead, make a new interface for rods and have a member attribute on it for the type of Pokemon caught.